cdap-ui/app/cdap/components/AbstractWidget/SqlSelectorWidget/SchemaContainer.tsx

```diff
@@ -0,0 +1,220 @@
+    });
+  };
+
+  public handleMenuOpen = (event) => {
+    this.setState({ menuAnchor: event.currentTarget });
+  };
+
+  public constructNewStage = (newSchema) => {
+    return {
+      ...this.props.stage,
+      schema: newSchema,
+    };
+  };
+
+  public toggleExpansionPanel = () => {
+    this.props.onExpandClick(this.props.stage);
+  };
+
+  public updateField = (newField) => {
+    if (this.props.stage && this.props.stage.schema) {
+      const newSchema = this.props.stage.schema.map((field) => {
+        if (field.name === newField.name) {
+          return newField;
+        }
+        return field;
+      });
+      this.props.onSchemaChange(this.constructNewStage(newSchema));
+    }
+  };
+
+  public render() {
+    const { classes } = this.props;
+    return (
+      <ExpansionPanel
+        className={classes.schemaContainer}
+        expanded={this.props.stage.expanded}
+        onChange={this.toggleExpansionPanel}
+      >
+        <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
+          <Typography variant="h6">{this.props.stage.name}</Typography>
+          <If condition={Boolean(this.props.errorCount)}>
+            <span className={classnames('badge', classes.badgeDanger)}>
+              {this.props.errorCount}
+            </span>
+          </If>
+        </ExpansionPanelSummary>
+        <ExpansionPanelDetails>
+          <Table>
+            <TableHead>
+              <TableRow>
+                <TableCell className={classes.schemaTableHeader} align="center">
+                  <Typography variant="h6">Name</Typography>
+                </TableCell>
+                <TableCell align="center" className={classes.schemaTableHeader}>
+                  <Button size="small" variant="outlined" onClick={this.handleMenuOpen}>
+                    <span>Select</span>
+                    <span
+                      className={classnames('fa', 'fa-chevron-down', classes.tableHeaderSelectIcon)}
+                    />
+                  </Button>
+                  <Menu
+                    anchorEl={this.state.menuAnchor}
+                    open={Boolean(this.state.menuAnchor)}
+                    onClose={this.handleMenuClose}
+                  >
+                    <MenuItem onClick={() => this.handleMenuClose('All')}>All</MenuItem>
+                    <MenuItem onClick={() => this.handleMenuClose('None')}>None</MenuItem>
+                  </Menu>
+                </TableCell>
+                <TableCell align="center" className={classes.schemaTableHeader}>
+                  <Typography variant="h6">Alias</Typography>
+                </TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {this.props.stage.schema &&
+                this.props.stage.schema.map((field, i) => {
+                  const aliasError = this.props.aliases && this.props.aliases[field.alias] > 1;
+                  // Material UI table styles has border on table head which is
```

Comment:
I think what this might have caused is the other rows in the table has its border top cut off.  It looks like this, 
![Screen Shot 2019-07-24 at 1 40 56 PM](https://user-images.githubusercontent.com/1452845/61827088-ae67d800-ae18-11e9-8bfe-005bc59a85f3.png)